### PR TITLE
Unique value_V1.0

### PR DESCRIPTION
**Pull request by divya2003**
Iss baar shi kiya h.. :-)

### CODE REVIEW

File: login/StoreUniqueValue [hunk 1]
```diff
@@ -1 +1,3 @@
-#store value of the unique field
\ No newline at end of file
+#store value of the unique field
+
```
Comment (jainsameer1991) [inline login/StoreUniqueValue]: ye file kya kar rhi hai?

Comment (divya2003) [reply login/StoreUniqueValue]: mene ek method create kiya h usme.

File: login/src/test/java/Actions/Fields.java [hunk 2]
```diff
@@ -56,5 +56,6 @@ public static void verifyRecordCreatedMessage(String objectName) {
 
 	public static void storeTextInVariable(String objectName) {
 		String storeTextString = getWebElement("").getText();
+		System.out.println("I am in love with Sameer");
```
Comment (jainsameer1991) [inline login/src/test/java/Actions/Fields.java]: @divya2003 What is this line doing?

Comment (divya2003) [reply login/src/test/java/Actions/Fields.java]: this line is printing an statement that I am in love with sameer jain.

File: login/StoreUniqueValue (hunk 1 above)
Comment (jainsameer1991) [reply login/StoreUniqueValue]: Can u plzz explain usme?

Comment (divya2003) [reply login/StoreUniqueValue]: its just a demo method I have created to understand git process. Please proceed with any further delay

**Pull request merged**